app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,7 @@
         </activity>
         <activity android:name=".ProfileActivity"></activity>
         <activity android:name=".EditProfile"></activity>
+        <activity android:name=".PictureDisplay"></activity>
```

Comment:
Always name your activity classes with "Activity" at the end. Same for fragments, service and so.
in this case like PictureActivity